ArgoProject/src/components/Collision.h

```diff
@@ -23,6 +23,8 @@ namespace app::comp
 	public: // Public Static Variables
 	public: // Public Member Variables
 		std::variant<cute::c2AABB, cute::c2Circle> bounds;
+		//if this is true entities should pass through everything
+		bool passable = false;
```

Comment:
For entities that still need to detect a collision but do not get pushed back (disc when its coming back towards player.  Otherwise anything with a impenetrable component would stop it. now instead there's an extra check 
if (!collision.passable)
{
       location.position += pushback;
}

Reply:
Then that should be part of disc attack, not the collision component as it is specific to the disc attack case.